src/Icicle/Avalanche/Statement/Flatten/Exp.hs

```diff
@@ -63,12 +65,14 @@ flatX a_fresh xx stm
      $ betaToLets a_fresh xx
 
   -- Annotation plumbing
-  flatX'    = flatX    a_fresh
-  makeApps' = makeApps a_fresh
-  xVar      = XVar     a_fresh
-  xPrim     = XPrim    a_fresh
-  xValue    = XValue   a_fresh
-  xApp      = XApp     a_fresh
+  flatX'           = flatX        a_fresh
+  Flat.FlatOps{..} = Flat.flatOps a_fresh
```

Comment:
I guess record wildcards only save a single argument here, but I do think it's a bit cleaner